# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: robot_arm_base_link__robot_arm_turning_mount
  pid: {p: 50, i: 50, d: 50}

joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: robot_arm_turning_mount__robot_arm_link2
  pid: {p: 50, i: 50, d: 50}

joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: robot_arm_link2__robot_arm_link3
  pid: {p: 50, i: 50, d: 50}

joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: robot_arm_link3__robot_arm_link4
  pid: {p: 50, i: 5, d: 50}